test/models/retrieveModels.m

```diff
@@ -0,0 +1,47 @@
+% retrieve models required for testing the CI that are published, but released as part of the COBRA Toolbox
+% eventual enhancement with rsync possible
+
+pth = which('retrieveModels.m');
+cd(pth(1:end-(length('retrieveModels.m')+1)));
+
+fprintf('\nDownloading models ...\n');
+
+modelArr = {
+    'iIT341.xml', 'http://bigg.ucsd.edu/static/models/iIT341.xml';
+    'Abiotrophia_defectiva_ATCC_49176.xml', 'https://webdav-r3lab.uni.lu/public/msp/AGORA/sbml/Abiotrophia_defectiva_ATCC_49176.xml';
+    'Ec_iJR904.xml', 'http://systemsbiology.ucsd.edu/sites/default/files/Attachments/Images/InSilicoOrganisms/Ecoli/Ecoli_SBML/Ec_iJR904.xml';
+    'Sc_iND750_flux1.xml', 'http://gcrg.ucsd.edu/sites/default/files/Attachments/Images/InSilicoOrganisms/yeast/Sc_iND750_flux1.xml';
+    };
+
+for i = 1:length(modelArr)
+    if exist(modelArr{i,1}, 'file') ~= 2
+        urlwrite(modelArr{i,2}, modelArr{i, 1});
```

Comment:
For now, we will not have such features but let's keep in mind that there is a tiny probability that these models may change in the future.